src/components/card/SmallPetCard.tsx

```diff
@@ -0,0 +1,82 @@
+import { UtcStringToYearMonth } from "@/utils/dateConverter";
+import { Icon } from "@iconify/react";
+import { useMemo } from "react";
+import { Link } from "react-router-dom";
+import BlankPic144x144 from "../../assets/small-petcard/BlankPic144x144.svg";
+interface SmallPetCardProps {
+  id: string;
+  name: string;
+  species: string;
+  gender: string;
+  birthdate: string;
+  status: string;
+}
+const smallpetcard = ({
+  id,
+  name,
+  species,
+  gender,
+  birthdate,
+  status,
+}: SmallPetCardProps) => {
+  const genderAns = useMemo(() => {
+    return gender === "male" ? "ผู้" : "เมีย";
+  }, [gender]);
+
+  const linkTo = useMemo(() => {
+    return "/pet/" + id;
+  }, [id]);
+  const { years, months } = useMemo(() => {
+    return UtcStringToYearMonth({ utcString: birthdate });
+  }, [birthdate]);
+  return (
+    <Link to={linkTo}>
+      <div className="m-0 h-64 w-44 shrink-0 rounded-2xl bg-white p-3 shadow-md">
+        <div className="flex flex-shrink-0 flex-col items-center justify-center gap-y-2.5">
+          <div className="flex w-36 rounded-2xl shadow-lg">
+            <img className="rounded-2xl" src={BlankPic144x144} alt={name} />
+          </div>
+
+          <div className="flex-start m-0 flex w-full flex-col items-start justify-center gap-2 p-0">
+            <div className="text-base font-semibold text-black">
+              {name} พันธุ์{species}
+            </div>
+            <div className="inline-flex w-full flex-row items-end justify-between">
```

Comment:
no need this div เอาออกไปเลยก็ได้ มันไม่ได้มีผลอะไร เพราะ Element วางเรียงมาอยู่แล้ว

Reply:
ผมมองก้อนล่างเป็น อีกก้อนใหญ่ ๆ ก้อนนึงง่ะ ละในนั้นก็ค่อยแยกเป็นส่วนซ้ายที่เป็นข้อความ ละส่วนขวาที่เป็นตัวติ๊กถูกง่ะ